src/php/views/page.twig

```diff
@@ -3,6 +3,12 @@
 {% block content %}
 
 	<div class="obj-width-limiter">
+		<img
+			src="{{ post.thumbnail.src|resize(1024) }}"
```

Comment:
Thank you for including the resize! 🏆 

It looks like `obj-width-limiter` uses `min` to ensure that the width does not exceed 60rem/960px. So I think we can update the value to `960`, unless there was a specific reason for `1024`.

Reply:
I think using a hardcoded resize value is perfect for this PR. There is also an opportunity to leverage the `picture`/`source` tags for responsive images, but I will create a separate issue for that.